### DISCUSSION TITLE
Controlnet Not Showing in UI

### DISCUSSION THREAD

**Discussion by DrewPear309** [AbdBarho/stable-diffusion-webui-docker#478] [General]
I have an empty folder in the root folder called controlnet and another at data\config\auto\extensions\ControlNet which is full of files.
I'm running on my CPU. I love this docker container. It renders 512x512 in 90 seconds. I've tried many other non-docker installations of automatic 1111 and it's variants. They all take around 210 seconds for the same render. that's huge. So i would love to get controlnet working. Also with this container, i can install extensions i never could before always getting an error about, "due to command line flags you're SOL" I figured it was because i was using my CPU. Controlnet always installed and worked fine though on all the other distros.
Thank you!

**Comment by zemmyang** [comment]
I looked for sd-webui-controlnet in the Extensions > Available tab, that made it work for me.
